winch/codegen/src/codegen/context.rs

```diff
@@ -182,31 +182,35 @@ impl<'a> CodeGenContext<'a> {
     /// range defined by the caller.  This is a specialization of the
     /// spill function; made available for cases in which spilling
     /// locals is not required, like for example for function calls in
-    /// which locals are not reachable by the callee.
+    /// which locals are not reachable by the callee.  It also tracks
+    /// down the amount of memory entries in the given range.
     ///
-    /// Returns the amount of registers spilled for the given range.
-    pub fn spill_regs_in<M, R>(&mut self, masm: &mut M, range: R) -> u32
+    /// Returns the amount of registers spilled and the amount of
```

Comment:
"Amount of registers" -> "number of registers"? (register is a count-noun so the "amount of" reads ambiguously to me: in bytes, or whole registers, or ...)